README.md

```diff
@@ -13,10 +13,12 @@
 
 # CyberArk Secrets Provider for Kubernetes
 
-The CyberArk Secrets Provider for Kubernetes enables you to use secrets stored and managed in the CyberArk Vault 
-using DAP or Conjur and consume them in your Openshift application containers. To do so, the CyberArk Secrets 
-Provider for Kubernetes image runs as an init container and provides the Conjur secrets, required by the pod, 
-from DAP or Conjur.
+The CyberArk Secrets Provider for Kubernetes enables DAP to retrieve secrets stored and managed in the CyberArk Vault. The
+ secrets can be consumed by your Kubernetes or Openshift application containers. To retrieve the secrets from Conjur or DAP, 
+ the CyberArk Secrets Provider for Kubernetes runs as an init container and fetches the secrets that the pods require. 
+ 
+ To do so, the CyberArk Secrets Provider for 
+Kubernetes runs as an init container and fetches secrets that pods require from DAP or Conjur.
```

Comment:
this text has some duplications. Did it go by a TW?

Reply:
that was removed in the previous force push ^